mscore/pm.cpp

```diff
@@ -18,6 +18,11 @@
 //  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 //=============================================================================
 
+#include "musescore.h"
+#include "preferences.h"
+#include "pm.h"
+#include "seq.h"
+
```

Comment:
I suppose this is not true for the same reason that "all.h" is automatically included...